.sync.yml

```diff
@@ -0,0 +1,52 @@
+---
+
+mock_with: ':rspec'
+
+.gitlab-ci.yml:
```

Comment:
I'd just remove gitlab-ci and appveyor since we won't ever use them on this module:
https://github.com/MiamiOH/hiera-ssm-paramstore/blob/master/.sync.yml#L1